monitor/basic/peer_monitor_test.go

```diff
@@ -33,7 +33,7 @@ func (mf *metricFactory) newMetric(n string, p peer.ID) api.Metric {
 	defer mf.l.Unlock()
 	m := api.Metric{
 		Name:  n,
-		Peer:  p,
+		Peer:  peer.IDB58Encode(p),
```

Comment:
```suggestion
		Peer:  p.Pretty(),
```

Reply:
No, IDB58Encode will probably always produce B58 encoding of the peer. Pretty() is subject to people whims on what 'pretty' means. Even the output produced by `.String()` has changed recently, without notice. Same is going to happen for `cid.String()` which will probably change to be base32.